### ISSUE TITLE
The ability to remove global styling?

### ISSUE THREAD

**Issue by disjointed** [LouisMazel/maz-ui#45]
**Is your feature request related to a problem? Please describe.**
Hello. I'd like to use some of your library's components in my current project, mainly inputs (text, date, phone). The project itself is not based on `moz-ui` and when I import, say, `<MozInput/>` inside my page (I'm using Nuxt.js) these styles gets added to body/html:
```
font: 400 16px -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
line-height: 1.5;
-webkit-font-kerning: normal;
font-feature-settings: "kern";
font-kerning: normal;
color: #212121;
background-color: white;
```
, which breakes my current template styling.

**Describe the solution you'd like**
Maybe these styles should be added to each component separately? Or have some component, like `<MozWrapper/>`, so when you put any other components inside these styles will apply to them?

Thank you in advance.

**Comment by LouisMazel** [comment]
Hi @disjointed,

Thank you so much for your feedback ! 

I just fixed and released the fix in the version `1.2.2`

Now only the font-size (`16px`) is set on the `<html />` element. I need it to set all components size with this base.

You can change this value like this: 

```scss
// main.scss

$base-font-size:    14px;
```
See the documentation: [How to change components colors and text style ?](https://louismazel.github.io/maz-ui/#/documentation/theme) 

---

**RELEASE 1.2.2 - Changelog**: https://github.com/LouisMazel/maz-ui/releases/tag/v1.2.2

**Comment by disjointed** [comment]
Well, that was fast! Thank you very much 👍
